contracts/sol6/reserves/bridgeReserve/uniswap/KyberUniswapReserve.sol

```diff
@@ -0,0 +1,335 @@
+pragma solidity 0.6.6;
+
+import "@uniswap/v2-core/contracts/interfaces/IUniswapV2Factory.sol";
+import "@uniswap/v2-core/contracts/interfaces/IUniswapV2Pair.sol";
+import "@uniswap/v2-periphery/contracts/interfaces/IUniswapV2Router01.sol";
+
+import "../../../IKyberReserve.sol";
+import "../../../IERC20.sol";
+import "../../../utils/WithdrawableNoModifiers.sol";
+import "../../../utils/Utils5.sol";
+import "../../../utils/zeppelin/SafeERC20.sol";
+
+contract KyberUniswapv2Reserve is IKyberReserve, WithdrawableNoModifiers, Utils5 {
```

Comment:
Better to use normal Withdrawable, and modifiers here

Reply:
yes
seems better
we don't have this file though.
so need to write it.